cmd/generate-rest-model/generate-rest-model.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"flag"
+	"io/ioutil"
+
+	"github.com/99designs/gqlgen/codegen/config"
+	"github.com/evergreen-ci/evergreen/rest/model"
+	"github.com/mongodb/grip"
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v2"
+)
+
+func main() {
+	var (
+		configFile string
+		schemaFile string
+		modelFile  string
+		helperFile string
+	)
+	flag.StringVar(&configFile, "config", "", "path to the config yml file with struct mappings")
+	flag.StringVar(&schemaFile, "schema", "", "path to the graphql schema to use for the output")
+	flag.StringVar(&modelFile, "model", "", "file path to write the generated model code <TODO relative>")
```

Comment:
wait sorry is this TODO for you or are we supposed to understand it

Reply:
doh that was a reminder to me to update the help text, which I definitely didn't forget...